content/faq/validations/host-format.adoc

```diff
@@ -0,0 +1,20 @@
+---
+title: Which formats does Kiali support for specifying hosts?
+weight: 70
+---
+:icons: font
+
+Istio highly recommend to always use FQDN for hosts on DestinationRules. However it allows to use other formarts like short names (_details_ or _details.bookinfo_).
+Kiali only supports FQDN and simple service name as host formats: _details.bookinfo.svc.cluster.local_ or _details_ instead. Validations using the _details.bookinfo_ format might not be accurate.
+
+In the following example it should show the validation of "More than one Destination Rule for the same host subset combination". Because of the usage of the short name _reviews.bookinfo_ Kiali won't show the warning message on both destination rules.
+
+[source, yaml]
+----
+include::/data/files/validation_examples/001-format-unsupported.yaml[]
+----
+
+See the recomendation Istio gives regarding https://istio.io/docs/reference/config/networking/destination-rule/#DestinationRule[host format^]:
+_"To avoid potential misconfigurations, it is recommended to always use fully qualified domain names over short names."_
```

Comment:
"To avoid potential misconfigurations, you should always use fully qualified domain names over short names."_

Reply:
This is something copy-pasted from istio documentation. So I'd rather leave it as it is in the documentation, just in case.